manifests/r/RustDesk/RustDeskNightly/1.2.4/RustDesk.RustDeskNightly.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: RustDesk.RustDeskNightly
+PackageVersion: 1.2.4
+InstallerType: exe
+Scope: machine
+InstallerSwitches:
+  Silent: --silent-install
+  SilentWithProgress: --silent-install
+  Interactive: --install
+UpgradeBehavior: install
+ReleaseDate: 2024-01-05
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.2.4-x86-sciter.exe
+  InstallerSha256: E64A1792446002F13628CEDF1BB0517A0E4A6FFA8799028D25A2F304A6117D44
+- Architecture: x64
+  InstallerUrl: https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.2.4-x86_64.exe
+  InstallerSha256: C9B40A905F3E778E6BD7646039CF2C8077C13E6FBBF74A70080884A5C23C780B

```

Comment:
At this time the hash is sadly outdated

Reply:
@OfficialEsco they update nightly builds once a day with all the latest commits

Reply:
Ye which could be problematic if there is delay with the bot or the mods ^^